Formula/crystal-lang.rb

```diff
@@ -6,25 +6,8 @@ class CrystalLang < Formula
   head "https://github.com/crystal-lang/crystal.git"
 
   stable do
```

Comment:
the stable do block is no longer needed since the patches are being removed, so the url and sha256 can be moved right above the head url again